com.unity.render-pipelines.high-definition/Editor/RenderPipeline/VirtualTexturingSettingsUI.cs

```diff
@@ -0,0 +1,267 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEditorInternal;
+using UnityEngine;
+using UnityEngine.Experimental.Rendering;
+using UnityEngine.Rendering.VirtualTexturing;
+
+#if ENABLE_VIRTUALTEXTURES
```

Comment:
Why is the UI for VT settings guarded here? #if ENABLE_VIRTUALTEXTURES
Just greying out the settings should be sufficient no ?

Reply:
There's no particular reason why we didn't show the settings when VT is disabled.
I've pushed a new commit that changes this to have the settings grayed out. Seems to work correctly but will notify QA about this as well.